.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -1,27 +1,35 @@
 name: 🐛 Bug report
 description: Submit a bug report to help us make Robocorp better
-title: "[Issue]: "
+labels: [ bug ]
 body:
-  - type: checkboxes
-    id: library
+  - type: dropdown
+    id: components
```

Comment:
Libraries aren't the only things we can improve in this repo, there's also the Action Server alone, integration tests, chores or dev utils. (and what knows more)